bidwire/scrapers/boston_notice_scraper.py

```diff
@@ -0,0 +1,64 @@
+import re
+import logging
+import scrapelib
+from document import Document, get_new_urls
+from datetime import datetime
+from datetime import date as dtdate
+from .base_scraper import BaseScraper
+from lxml import html
+from concurrent.futures import ThreadPoolExecutor
+
+
+# Logger object for this module
+log = logging.getLogger(__name__)
+
+
+class BostonNoticeScraper(BaseScraper):
+    NOTICES_URL = "https://www.boston.gov/public-notices"
+
+    def __init__(self, threads=4):
+        self.scraper = scrapelib.Scraper()
+        self.thread_executor = ThreadPoolExecutor(threads)
+
+    def scrape_desc(self, href):
+        desc_page = self.scraper.get('https://www.boston.gov' + href)
+        tree = html.fromstring(desc_page.content)
+        try:
```

Comment:
This is not functionally correct. As per the story description in https://www.pivotaltracker.com/story/show/145036889, "the text below each link is the italicized subtitle on the corresponding Public Notice page."

Looking at the HTML of a sample notice, seems like you should be looking at the plain text inside a `div` with `class=intro-text`.